### PR TITLE
Pre-Commit: Python Formatting

### PR DESCRIPTION
**Pull request by ax3l**
Add automatic Python style formatting using ["black"](https://black.readthedocs.io).
That way, we don't need to worry ever about formatting questions and have a consistent style everywhere.

### CODE REVIEW

File: examples/iota_lattice/run_iotalattice.py [hunk 1]
```diff
@@ -99,12 +98,63 @@
...
+    drd4,
+    edge30,
+    sbend30,
+    edge30,
+    dre1,
+    qe1,
+    dre2,
+    qe2,
+    dre3,
 ]
```
Comment (ax3l) [inline examples/iota_lattice/run_iotalattice.py:108]:
```suggestion
]
#   fmt:on
```

File: examples/iota_lattice/run_iotalattice.py [hunk 2]
```diff
@@ -99,12 +98,63 @@
 
 # build lattice: first half, qe3, then mirror
 lattice_half = [
```
Comment (ax3l) [inline examples/iota_lattice/run_iotalattice.py:101]:
```suggestion
#   fmt: off
lattice_half = [
```

File: examples/iota_lattice/run_iotalattice.py [hunk 3]
```diff
@@ -98,14 +97,67 @@
...
+    qd4,
+    drd4,
+    edge30,
+    sbend30,
+    edge30,
+    dre1,
+    qe1,
+    dre2,
+    qe2,
+    dre3,
```
Comment (ax3l) [inline examples/iota_lattice/run_iotalattice.py]:
```suggestion
#   fmt: off
lattice_half = [
    dra1, qa1, dra2, qa2, dra3, qa3, dra4, qa4, dra5,
    edge30, sbend30, edge30, drb1, qb1, drb2, qb2, drb2, qb3,
    drb3, dnll, drb3, qb4, drb2, qb5, drb2, qb6, drb4,
    edge60, sbend60, edge60, drc1, qc1, drc2, qc2, drc2, qc3, drc1,
    edge60, sbend60, edge60, drd1, qd1, drd2, qd2, drd3, qd3, drd2, qd4, drd4,
    edge30, sbend30, edge30, dre1, qe1, dre2, qe2, dre3
```

**Review by n01r — APPROVED** [review]
Awesome! Thank you so much! :)

**Review by cemitch99 — APPROVED** [review]
Ok.

**Pull request merged**